src/Glpi/Dashboard/Widget.php

```diff
@@ -1543,39 +1517,44 @@ function(param) {
             ];
         }
 
-        $options_json = Json::encode(
-            $options,
-            false,
-            ['enableJsonExprFinder' => true]
-        );
-
-        $js = <<<JAVASCRIPT
-        $(function () {
-            if (GLPI.Dashboard.getActiveDashboard()) {
-                var target = GLPI.Dashboard.getActiveDashboard().element.find('#{$chart_id} .chart');
-            } else {
-                var target = $('#$chart_id .chart');
-            }
-            var myChart = echarts.init(target[0]);
-            myChart.setOption($options_json);
-            myChart
-                .on('click', function (params) {
-                    const data_url = _.get(params, 'data.url', '');
-                    if (data_url.length > 0) {
-                        window.location.href = data_url;
+        $twig_params = [
+            'chart_id' => $chart_id,
+            'options' => $options,
+            'show_points' => $p['show_points'],
+            'point_labels' => $p['point_labels'],
+        ];
+        // language=Twig
+        $js = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+            <script type="module">
+                const target = GLPI.Dashboard.getActiveDashboard() ?
+                    GLPI.Dashboard.getActiveDashboard().element.find('#{{ chart_id }} .chart')
+                    : $('#{{ chart_id }} .chart');
+                const chart_options = {{ options|json_encode|raw }};
+                
+                $.each(chart_options.series, function (index, serie) {
+                    if ({{ show_points ? 'true' : 'false' }}) {
+                        serie['symbol'] = (param) => param > 0 ? 'circle': 'none';
```

Comment:
Variable name was `value` in the previous code. I guess it matches the component documentation.
```suggestion
                        serie['symbol'] = (value) => value > 0 ? 'circle': 'none';
```